green_city/tests/api/get_eco_news_likes_test.py

```
@@ -0,0 +1,88 @@
+
+def test_get_eco_news_likes_count_response_headers(create_news):
+    eco_news_id = create_news
+
+    response = requests.get(
+        f"{API_BASE_URL_8085}{ENDPOINTS['create_eco_news']}/{eco_news_id}/likes/count"
+    )
+
+    assert response.status_code == 200
+
+    headers = response.headers
+    assert "x-content-type-options" in headers.keys()
```

Ruff (0.11.9)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)